docs/UserGuide.md

```diff
@@ -145,6 +145,73 @@ In this section, you will learn how to set up LinkMeIn in your computer.
 
 # Using LinkMeIn's CLI
 
+LinkMeIn uses the Command Line Interface (CLI). 
+CLI is a means of interacting with a computer program by **inputting lines of text called command-lines**.
+
+Do not be put off by its name! Read on to find out how simple CLI is in LinkMeIn.
+
+_Read a quick [introduction](#introducing-linkmeins-gui) to familiarise yourself with our user interface._
```

Comment:
Can don't need this? since they would have already read thru the previous section.